playground/src/pages/CodingEnvironment.tsx

```diff
@@ -134,14 +142,16 @@ export default function CodingEnvironment(): JSX.Element {
 "trace_len" : ${trace_len}
 }`);
         }
-        // Store the proof if >0 (empty proof is 0)
+        
+        toast.success(`Proving successful in ${Date.now() - start} ms`, {id: 'provingToast'});
```

Comment:
please also add the timing to the verify step

Reply:
ok